swift_browser_ui_frontend/src/common/globalFunctions.js

```diff
@@ -31,6 +31,16 @@ export function toggleCopyFolderModal(folderName, sourceProjectId) {
   modifyBrowserPageStyles();
 }
 
+export function toggleDeleteModal(objects, containerName) {
+  store.commit("toggleDeleteModal", true);
+  if (objects) {
+    store.commit("setDeletableObjects", objects);
+  }
+  if (containerName) {
+    store.commit("setFolderName", containerName);
+  }
+}
```

Comment:
I think we could also add `modifyBrowserPageStyles()` for functions that handle opening/closing DeleteModal. 
This one helps prevent scrolling in the background when a modal is opened :)

Reply:
I've been wondering what was it for

Reply:
I see. We could fix this jumping to the top of the page in another PR :)

Reply:
This PR fixes the jumping issue: https://github.com/CSCfi/swift-browser-ui/pull/1024